src/common/records.lua

```diff
@@ -7,27 +7,41 @@ Records = Records
 		["@"] = {
 			transactionId = "-k7t8xMoB8hW482609Z9F4bTFMC3MnuW8bTvTyT8pFI",
 			ttlSeconds = 900,
+			priority = nil,
```

Comment:
let's default it to `0`

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/77/commits/a32e90a50330ea185fc2469eb1f1f4d574e9b791